int8   calib_state

### calibration state enum
int8   UNCALIBRATED = 0
int8   CALIBRATING  = 1
int8   CALIBRATED   = 2

